nav2_behavior_tree/include/nav2_behavior_tree/plugins/condition/is_path_valid_condition.hpp

```diff
@@ -68,6 +68,7 @@ class IsPathValidCondition : public BT::ConditionNode
   // The timeout value while waiting for a responce from the
   // is path valid service
   std::chrono::milliseconds server_timeout_;
+  bool first_time_;
```

Comment:
I would suggest keeping the pattern you have established with all the other nodes. 

`bool initialized`
`initialized()`

Reply:
done